.github/workflows/reusable-create-java-instrumentation-pull-request.yml

```diff
@@ -0,0 +1,63 @@
+name: Reusable - Create Java instrumentation pull request
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        type: string
+        required: true
+    secrets:
+      BOT_TOKEN:
+        required: true
+  # to help with partial release build failures
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version"
+        required: true
+
+jobs:
+  create-java-instrumentation-pull-request:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: opentelemetrybot/opentelemetry-java-instrumentation
+          # this is the personal access token used for "git push" below
+          token: ${{ secrets.BOT_TOKEN }}
+
+      - name: Initialize pull request branch
+        env:
+          VERSION: ${{ inputs.version }}
+        run: |
+          git remote add upstream https://github.com/open-telemetry/opentelemetry-java-instrumentation.git
+          git fetch upstream
+          git checkout -b update-opentelemetry-sdk-to-${VERSION} upstream/main
+
+      - name: Update version
+        env:
+          VERSION: ${{ inputs.version }}
+        run: ./.github/scripts/update-sdk-version.sh $VERSION
```

Comment:
This script doesn't exist yet. Can you link to the PR when it's available?

Reply:
merged it just now: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/6433